framework/src/modules/chain/blocks/blocks.js

```diff
@@ -17,6 +17,7 @@
 const EventEmitter = require('events');
 const { cloneDeep } = require('lodash');
 const blocksUtils = require('./utils');
+const { convertErrorsToString } = require('../helpers/error_handlers');
```

Comment:
No need to use this anymore, our new logger can put object as first parameter

Reply:
Applied your suggestions, please give your input on `framework/src/modules/chain/blocks/blocks.js:777`